packages/babel-core/test/config-chain.js

```diff
@@ -1060,30 +1090,33 @@ describe("buildConfigChain", function () {
         );
       });
 
-      test.each(
-        [
-          "babel.config.json",
-          "babel.config.js",
-          "babel.config.cjs",
-          // We can't transpile import() while publishing, and it isn't supported
-          // by jest.
-          process.env.IS_PUBLISH ? "" : "babel.config.mjs",
-        ].filter(Boolean),
-      )("should load %s asynchronously", async name => {
+      test.each([
+        "babel.config.json",
+        "babel.config.js",
+        "babel.config.cjs",
+        "babel.config.mjs",
```

Comment:
nit: The reason I filtered `.mjs` in the test cases is that it will not print
```
✔️ should load babel.config.mjs asynchronously
```
on node.js 8 because apparently we do not support that. If we add to the test cases and then skip it (treat it as successful), it can be confusing to new contributors.

Reply:
It still warns `Skipping "should load babel.config.mjs asynchronously" because native ECMAScript modules are not supported`. I agree that it's not ideal, but I think it's better to have the tests structured in a way that it's easier to think about them (`.filter` doesn't always work, because we sometimes use the template form of `jest.each`).

Also, that possible confusion only affects users running Node.js 10: older versions aren't supported by Jest, and newer versions support `.mjs` files.